Client/Frontend/Browser/TabTrayControllerV1.swift

```diff
@@ -827,92 +827,6 @@ fileprivate class TabLayoutDelegate: NSObject, UICollectionViewDelegateFlowLayou
     }
 }
 
-private struct EmptyPrivateTabsViewUX {
```

Comment:
Moved the Empty private tabs view to be used in both V1 and V2 version of ViewController